server/routes/reviewsTable.ts

```diff
@@ -78,7 +78,8 @@ export default function routes(router: Router): Router {
       order,
     })
 
-    const pageCount = Math.ceil(response.reviewCount / PAGE_SIZE)
+    const { reviewCount } = response.levels.find(level => level.levelCode === selectedLevelCode)
```

Comment:
a level can have 0 prisoners/reviews and from what i can read here it'd render ok… right?

Reply:
Yeah, if there are no prisoners at a given level `reviewCount` would be `0` which means `pageCount` would be `1` (ceil(`0/20`) => ceil(`0`))